scripts/cluster/start.sh

```diff
@@ -57,13 +57,15 @@ if [ -z "${GSA_EMAIL}" ]; then
   done
 fi
 
-if ! kubectl get secret/gcsfuse-secrets --namespace kube-system; then
-  KEY_PATH=$(mktemp -d)/key.json
-  gcloud iam service-accounts keys create "${KEY_PATH}" --iam-account "${GSA_EMAIL}"
-  kubectl create secret generic gcsfuse-secrets --from-file="${KEY_PATH}" --namespace kube-system
-  rm -rf $(dirname "${KEY_PATH}")
+KSA_NAME="gcsfuse-sa"
+
+if ! kubectl get serviceaccount/${KSA_NAME} --namespace kube-system; then
+  kubectl create serviceaccount --namespace kube-system ${KSA_NAME}
```

Comment:
Is the if necessary? Can we use the apply format instead?

Reply:
See the way the configmap is created (it pipes a create into an apply)

Reply:
Yes, you are right. Changed for apply.